src/pages/index.hbs

```diff
@@ -4,7 +4,9 @@ layout: main.hbs
 head:
  - <link rel="stylesheet" href="en/<%= latest %>/css/ol.css" type="text/css">
  - <link rel="stylesheet" href="<%= assets %>/theme/index.css" type="text/css">
- - <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=requestAnimationFrame,Element.prototype.classList"></script>
+ - <script src="https://unpkg.com/elm-pep"></script>
+ - <script src="https://cdn.polyfill.io/v3/polyfill.min.js?features=fetch,requestAnimationFrame,Element.prototype.classList,TextDecoder"></script>
```

Comment:
Are all of these still needed?

Reply:
I think for the website it will be best to only include what we need, so the previous version should do fine for the polyfill.io polyfills.